psutil/_psutil_osx.c

```diff
@@ -860,13 +874,55 @@ psutil_boot_time(PyObject *self, PyObject *args) {
     return Py_BuildValue("f", (float)boot_time);
 }
 
+/*
+ * Return a Python float indicating the value of the temperature
+ * measured by an SMC key
+ */
+static PyObject *
+psutil_smc_get_temperature(PyObject *self, PyObject *args) {
+    char* key;
+    float temp;
```

Comment:
add new line

Reply:
Sorry, I don't understand. 
It seems to be the same format as e.g.
```c
/*                                                                                                                                                            
 * Return CPU statistics.                                                                                                                                     
 */                                                                                                                                                           
static PyObject *                                                                                                                                             
psutil_cpu_stats(PyObject *self, PyObject *args) {                                                                                                            
    struct vmmeter vmstat;          
```
Further in the same file. 
Do you want it to look like this?
```c
static PyObject *
psutil_smc_get_temperature(PyObject *self, PyObject *args) {

    char* key;
    float temp;
```

Reply:
NVM, saw you answer